### PR TITLE
NPC ship cleanups

### PR DESCRIPTION
**Pull request by caelaislinn**
Partially rewrites radio chatter and ship spawning. Should cut down on radio spam

### CODE REVIEW

**Review by BDpuffy420 — APPROVED** [review]

**Comment by caelaislinn** [comment]
Actually I have to mention... overmap radio comms are only partially working. I've got a rewrite in the works but it's pretty complex and won't be ready for a while.

File: maps/_gamemodes/invasion/gamemode.dm [hunk 1]
```diff
@@ -33,6 +33,9 @@
 
 	setup_objectives()
 
+	shipmap_handler.spawn_ship("Human Colony", 3)
+	shipmap_handler.spawn_ship("UNSC", 2)
+
```
Comment (X0-11) [inline maps/_gamemodes/invasion/gamemode.dm:38]: covvie ships too pls

Comment (X0-11) [reply maps/_gamemodes/invasion/gamemode.dm:38]: also readd the chance for a couple of flood ships to spawn

Comment (caelaislinn) [reply maps/_gamemodes/invasion/gamemode.dm:38]: I don't want any flood in this gamemode for now, and the covvies don't need any buffing. I'd like to keep NPC ship numbers low as well to keep down on processing and while they are still a little unstable

Comment (X0-11) [reply maps/_gamemodes/invasion/gamemode.dm:38]: Fair enough. I had wanted like 1-2 covvie ships just to fill the 1v1v1 up.

**Comment by caelaislinn** [comment]
#1727

**Review by BDpuffy420 — APPROVED** [review]
Oh yeah now it’s good!

**Comment by caelaislinn** [comment]
#960 727

**Comment by caelaislinn** [comment]
#1727

**Pull request merged**